liquid.py

```diff
@@ -214,14 +225,26 @@ def first(items, name_plural='items'):
     return items[0]
 
 
-def shell(name, *args):
+def shell_cmd(name, *args):
     """
     Open a shell in a docker container tagged with liquid_task=`name`
     """
     containers = docker.containers([('liquid_task', name)])
     container_id = first(containers, 'containers')
-    docker_exec_cmd = ['docker', 'exec', '-it', container_id] + list(args or ['bash'])
-    run_fg(docker_exec_cmd, shell=False)
+    args = [arg.replace('__nomad_address__', get_nomad_address()) for arg in args]
+    docker_exec_cmd = ['docker', 'exec', container_id] + list(args or ['bash'])
+    return run_fg(docker_exec_cmd, shell=False)
+
+
+def shell(name, *args):
+    """
+    Open a shell in a docker container tagged with liquid_task=`name`
+    """
+    try:
+        shell_cmd(name, *args)
+    except CalledProcessError as e:
+        print(e.output.decode(), file=sys.stderr)
```

Comment:
I needed to print the command output in case of error. Obsolete, as I changed shell to the original functionality.

Reply:
Then please remove the try/except.

Reply:
Actually I need to process the command output to read the list of collections from the hoover search index. In case something went wrong, I need to print the output of the command in this service. The CalledProcessError exception stacktrace does not contain that data.

Reply:
Then please use another function, perhaps `docker_exec`, and leave `shell` as simple as possible.

Reply:
But it is `docker_exec`, please take a look at the code, the snippet you see here is obsolete.